enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -81,7 +81,6 @@ export const _InteractiveQuestion = ({
       await dispatch(initializeQBRaw(location, params));
     } catch (e) {
       console.error(`Failed to get question`, e);
-    } finally {
```

Comment:
This is the fix, `finally` will run even when `initializeQBRaw` dosen't throw any error. This isn't enough to say everything is already loaded. The only thing that's guaranteed to be loaded by the time we this action returns is the question.